new_project_template/README.md

```diff
@@ -0,0 +1,21 @@
+# Template
+
+This is a folder with a template set of files to create a new project in hi-ml.
+
+To use it:
+
+* Copy the template folder to the desired project folder
+* In the new location, rename the folders `health_multimodal` and `test_multimodal` to be the desired Python namespaces
+* Search for all occurences of `multimodal` and replace with the desired project name. Don't use a simple query/replace
+  strategy but do check the context!
+* Move the workflow definition `new-project-pr.yml` to `.github/workflows` and give it a name that matches the project.
+
```

Comment:
Should we mention something about updating the *_requirements.txt files if necessary?

Reply:
yup.